website/src/hooks/useCopyToClipboard.js

```diff
@@ -0,0 +1,16 @@
+import { useCallback } from 'react'
+import { useToast } from '@aragon/ui'
+import { writeText as copy } from 'clipboard-polyfill'
+
+export function useCopyToClipboard () {
```

Comment:
Is this not already a part of aUI? We already have some similar functionality by using the `IdentityBadge` for organisations in the overview.

Reply:
Something like this is implemented but not exposed, and the way the AccountModule handles this is a bit different; eventually we will migrate this to aragonUI when we see another use case for this.